peekaboo/ruleset/rules.py

```diff
@@ -34,7 +34,7 @@
 from peekaboo.exceptions import PeekabooAnalysisDeferred, \
         CuckooSubmitFailedException, PeekabooRulesetConfigError
 from peekaboo.toolbox.ole import Oletools
-from peekaboo.toolbox.files import Filetools
+from peekaboo.toolbox.file import Filetools
```

Comment:
Any specific reason for this change?

Reply:
Addressed in #143 by spelling out that it's to be in alignment with singular use in other toolbox modules and because the module deals with multiple properties of individual files and not the other way around.